### DISCUSSION TITLE
Regarding current vulnerabilities in NGINX CVE-2023-5043/CVE-2022-4886/

### DISCUSSION THREAD

**Discussion by ThenmozhiBalaji** [nginx/kubernetes-ingress#4611] [Q&A] [answered]
Hi All,

  We are running nginxinc open source version we would like to know if above vulnerability is applicable here as well ?

https://github.com/kubernetes/ingress-nginx/issues/10571
https://github.com/kubernetes/ingress-nginx/issues/10570
https://github.com/kubernetes/ingress-nginx/issues/10572

Above talks about chrooted and annotation injection whereas nginxinc runs as 101 nginx user is above appicable ?. Btw we are using nginx for only udp traffic. It will help a lot if someone can share input on the same.

rgds
Balaji

**Comment by ThenmozhiBalaji** [comment]
Btw above are from 
https://thehackernews.com/2023/10/urgent-new-security-flaws-discovered-in.html?_m=3n%2e009a%2e3187%2emm0ao0eq25%2e26ei

**Reply by ciarams87** [reply, marked as answer]
Hi @ThenmozhiBalaji 

There is a blog post that covers this topic: https://www.nginx.com/blog/which-nginx-ingress-controllers-are-impacted-by-cve-2022-4886-cve-2023-5043-and-cve-2023-5044/

In short, these vulnerabilities _only_ apply to the community version.
